docs/competition/competition-roles.md

```diff
@@ -0,0 +1,55 @@
+# Competition Roles
+
+To run a successful competition, there are a number of roles which need to be assigned. One role may be filled by multiple volunteers, and one volunteer may be filling multiple roles. Some roles should only be assigned to a single volunteer, and this be their only role.
+
+Depending on the nature of the event, some of these roles may not be applicable.
+
+- [Intro Briefer](./event/intro-brief.md)
+- [Refreshments](./volunteers/refreshments.md)
+- [Team Pit Manager](./team-pits/README.md)
+- [Robot Safety Inspector](./matches/README.md#robot-safety-inspections)
+- Group Photo Photographer
+- Health and Safety Coordinator
```

Comment:
Link to the existing H&S docs? (I'm thinking of the ones Scarzy added)

Reply:
https://github.com/srobo/runbook/pull/114/commits/ba7ceae44f18a23333356eadb8b6973846e648d1